### ISSUE TITLE
Firefox Nightly 78.01a crashes when opening Sidebery Configuration Panel

### ISSUE THREAD

**Issue by SivaMarti** [mbnuqw/sidebery#229]
Right-Click on Sidebery panel and clicking Configuration Panel from drop down menu, opens the Configuration Panel, but then causes Firefox to crash. Firefox, upon restarting, tries to open where it left off due to options setting, which is Sidebery Configuration Panel. Firefox crashes again. The only way to stop the loop is to start Firefox in Safe mode, click on a different tab, and then close and reopen.

Sidebery Configuration Panel opens normally to change configuration options

Firefox Crashes

OS: Windows 10 
Firefox version:  Nightly 78.01a
Addon version:  4.8.0

<details>
I am unable to get this information because I can't bring up the settings page without Firefox crashing.</details>

**Comment by mbnuqw** [comment]
Thanks for the report. Unfortunately, I cannot reproduce this. Does this problem appear in Firefox Beta?

**Comment by SivaMarti** [comment]
I'm not sure. I only use Firefox nightly, but I will download Firefox Beta and test it. The odd thing is all other functions seem to be fine with Sidebery and Firefox Nightly. I've been using Sidebery since Firefox Nightly 70 and every upgrade I'm seen no issues. It's just trying to change settings now that I get a crash. If I leave things as they are Sidebery works no problem, but it's not optimal. I'm not sure if it's another add-on or my Sidebery CSS and I'm not sure exactly how to test the CSS. I'll test in Firefox Beta and report back.

**Comment by SivaMarti** [comment]
Thank you for the response previously by the way. I really appreciate this extension and am very happy with what it does. After I worked out how to initially configure the userChrome.css it's been a dream extension.

I installed Firefox Beta and synced my extensions. Sidebery works perfectly like expected, even with my userChrome.css copied into the profiles directory in the Chrome folder. From that test I don't believe the issue is in the userChrome.css file itself.

The only problem I am having is I'm trying to import the Sidebery data into the Firefox Beta from Nightly. When I click on import I don't know where the .json file would be for the Nightly profile. I want to test my configuration. Maybe there is a bug in the way I have it set up in Nightly.

Can you let me know where it should be stored so I can import it?

**Comment by mbnuqw** [comment]
Hm... this will be problematic, because to import Sidebery data to another profile you'll need first to export it from the origin profile (in settings page which cannot be reached in your case).

About the crashes - you can try to enable / disable WebRender in about:config ("gfx.webrender.all" flag), but this is just a shot in the dark...

**Comment by SivaMarti** [comment]
Thanks. I'll give it a try. Since it's working in Firefox Beta fine, it may be something specific to my install in Nightly. I may just continue to use Beta as my default for the moment and moved my tabs.

**Comment by SivaMarti** [comment]
As an update, I tried toggling gfx.webrender.all in about:config. Unfortunately it did not appear to change anything. It did not resolve the issue.

Because Sidebery was working as before in Firefox Beta I manually created my panels again which was not a huge deal and then made a backup of all my tabs and imported them into Firefox Beta. I just had to rearrange the tabs in the appropriate panels. Once I set Sidebery up again successfully and tested opening the configuration/settings I was sure to create a Sidebery backup this time.

I think there was something glitchy in my Nightly setup that was causing the issue. Thank you for your time.
